src/main/scala/org/broadinstitute/dsde/workbench/avram/dependencies/AvramDependencies.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.workbench.avram.dependencies
+
+import org.apache.commons.dbcp2.BasicDataSource
+import org.broadinstitute.dsde.workbench.avram.dataaccess.SamDao
+import slick.jdbc.PostgresProfile.api._
+
+case class AvramDependencies(database: Database,
```

Comment:
Container for all dependencies to "inject" into the `Reader` portion of `AvramResult`.